backend/models/feedbackModel.js

```diff
@@ -6,9 +6,22 @@ const feedbackSchema = mongoose.Schema(
       type: String,
       required: true,
     },
+    user_id: {
+      type: mongoose.Schema.Types.ObjectId,
+      required: true,
+    },
     task_id: {
       type: mongoose.Schema.Types.ObjectId,
+      required: true,
+    },
+    satisfaction: {
+      type: Number,
+      required: false
     },
+    createDate: {
+      type: Date,
```

Comment:
I don't think the `createDate` needs to be a required value here but should have a default value.
Similar to the way how a task is created and has a createdDate value

Reply:
In this case, if you already set a default value in here, then you won't even need to set it in the feedbackController

Reply:
Fixed: Changes are reflected in recent commits